src/main/assets/js/session-timeout.ts

```diff
@@ -0,0 +1,42 @@
+import _ from 'lodash';
```

Comment:
For webpack tree shaking you might want to change this to:

```suggestion
import { throttle } from 'lodash';
```

This way webpack should only bundle `throttle` to the frontend rather than including the whole of lodash.